### PR TITLE
DOC: Update VCS Principles Tutorial

### PR DESCRIPTION
**Pull request by davis278**
Deleted inappropriate code at the top of the page; made other corrections to the text.

### CODE REVIEW

File: Jupyter/EzPlot_Example/EzPlot_Example.html [hunk 1]
```diff
@@ -3,8 +3,6 @@
 title: EzPlot_Example Tutorial
 ---
 
-# EzPlot_Example Tutorial
```
Comment (doutriaux1) [inline Jupyter/EzPlot_Example/EzPlot_Example.html]: We want the download link to the notebook. Why did you remove it?

Comment (doutriaux1) [reply Jupyter/EzPlot_Example/EzPlot_Example.html]: Is the link broken?

File: contact.md [hunk 2]
```diff
@@ -64,9 +64,9 @@ To leave the mailing list, send an email to [listserv@listserv.llnl.gov][devunsu
 <a href="https://www.youtube.com/results?search_query=UVCDAT"><img class="contact-logo" src="/images/youtube.png" alt="YouTube logo"/></a>
 
 [uvcdat-askbot]:   http://askbot-uvcdat.llnl.gov
```
Comment (doutriaux1) [inline contact.md]: askbot is dead, can you please remove it. Thanks.

File: getting_started.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ source activate [YOUR_CDAT_CONDA_ENV]
 
 Once you've loaded the environment, you should be able to run the examples. They should output a .png file that has the same image as the example.
 
-We strongly recommend using Jupyter notebook for the tutotrials
+We strongly recommend using Jupyter notebook for the tutotrials.  When you type the command below it is best to have navigated to a folder that contains at least one jupyter notebook (file extention .ipynb).
```
Comment (doutriaux1) [inline getting_started.md]: typo on `tutorials` and actually in notebook you can't go up so it would say something like:
When you type the command below it is best to have navigated to a folder containing the jupyter notebooks sub-folders.

File: getting_started.md [hunk 4]
```diff
@@ -62,7 +62,7 @@ vcs.download_sample_data_files()
 # The vcs_canvas is the root object of VCS
 vcs_canvas = vcs.init()
 
-cdms_file = cdms2.open(vcs.prefix + "/sample_data/clt.nc")
+cdms_file = cdms2.open(vcs.prefix + "/share/cdat/sample_data/clt.nc")
```
Comment (doutriaux1) [inline getting_started.md]: really it should
```python
import os
import cdat_info
cdms_file = cdms2.open(os.path.join(cdat_info.get_sampledata_path(), "clt.nc"))
```

File: tutorials.html [hunk 5]
```diff
@@ -7,12 +7,20 @@
...
+    <p>Other learning resources include:
+        <ul><li><a href="https://www.dataquest.io/blog/jupyter-notebook-tutorial/">Dataquest's Jupyter Notebook Tutorial</a></li>
+            <li><a href="https://jupyter-notebook-beginner-guide.readthedocs.io/en/latest/">Read the Docs' Jupyter/IPython Notebook Quick Start Guide</a></li>
+            <li><a href="https://www.datacamp.com/community/tutorials/tutorial-jupyter-notebook">DataCamp's Jupyter Notebook Tutorial: The Definitive Guide</a></li>
+        </ul>
     </p>
     <div class="container">
         <h3>CDMS</h3>
-        <p> Example on how to use CDMS</p>
+        <p> Examples of how to use CDMS</p>
```
Comment (doutriaux1) [inline tutorials.html]: thx!

File: Jupyter/VCS_Principles/VCS_Principles.html [hunk 6]
```diff
@@ -3,8 +3,6 @@
 title: VCS_Principles Tutorial
 ---
 
-# VCS_Principles Tutorial
```
Comment (doutriaux1) [inline Jupyter/VCS_Principles/VCS_Principles.html]: we need the download link.

File: Jupyter/EzPlot_Example/EzPlot_Example.html (hunk 1 above)
Comment (davis278) [reply Jupyter/EzPlot_Example/EzPlot_Example.html]: It wasn't a link. It was just text that didn't do anything.  I fixed this problem for the EzPlot_Example Tutorial in #168 by deleting the useless text and adding in an html link to the notebook.

File: contact.md (hunk 2 above)
Comment (davis278) [reply contact.md]: Done.

File: getting_started.md (hunk 3 above)
Comment (davis278) [reply getting_started.md]: I've fixed the typo and good point on not being able to navigate to a higher folder after launching the Jupyter Notebook app. See if the way I've rephrased things makes sense now. If not, I can edit some more.

File: getting_started.md (hunk 4 above)
Comment (davis278) [reply getting_started.md]: In trying to test this out, I was not able to import vcs or cdms2 using the cdat8 environment mentioned in the installation instructions. @doutriaux1 if you have time next week, perhaps we can look at this together.

File: Jupyter/VCS_Principles/VCS_Principles.html (hunk 6 above)
Comment (davis278) [reply Jupyter/VCS_Principles/VCS_Principles.html]: The text I removed that looks like a download link for the notebook, was not an active link. It appears to be markdown code that did not get converted properly to html so it should be removed.
I did not delete the true link for the notebook which is included in the following html code:
<li><a href="VCS_Principles.ipynb">Download the Jupyter notebook</a></li>
Once the question about the example listed in the getting_started.md file has been resolved, @doutriaux1 could you review this pull request again in light of this explanation?

File: getting_started.md (hunk 4 above)
Comment (davis278) [reply getting_started.md]: Done.
